### DISCUSSION TITLE
I have a segmentation fault

### DISCUSSION THREAD

**Discussion by edsousa84** [trevorsandy/lpub3d#582] [Q&A] [answered]
Is there anything I can make to track the problem?
I am on a Ubuntu 18.04 machine

**Comment by trevorsandy** [comment, marked as answer]
Thank you for your question.

If this behaviour is still present with the latest release, you can provide the LPub3D output log.

Cheers,

**Comment by edsousa84** [comment]
I forgot about this, On new projects and on new versions  the problem did not appen again,
